.github/workflows/release.yml

```diff
@@ -0,0 +1,101 @@
+name: Release
+
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  regression:
+
+    name: Python 3.x on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-2016]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/cache@v1
+      if: startsWith(matrix.os, 'ubuntu')
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+
+    - uses: actions/cache@v1
+      if: startsWith(matrix.os, 'macOS')
+      with:
+        path: ~/Library/Caches/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+
+    - uses: actions/cache@v1
+      if: startsWith(matrix.os, 'windows')
+      with:
+        path: ~\AppData\Local\pip\Cache
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
```

Comment:
Regression uses `architecture: ${{ startsWith(matrix.os, 'windows') && 'x86' || 'x64' }}`, release defaults to `architecture: 'x64'` as it's not specified.

Reply:
it needs python >= 3.5 at least

Reply:
"3.x" is just "the latest", afaik